lib/runtime/src/pipeline/network/ingress/push_handler.rs

```diff
@@ -220,6 +221,14 @@ where
         let mut send_complete_final = true;
         while let Some(resp) = stream.next().await {
             tracing::trace!("Sending response: {:?}", resp);
+            if let Some(err) = resp.err() {
+                const STREAM_ERR_MSG: &str = "Stream ended before generation completed";
```

Comment:
Also can STREAM_ERR_MSG be defined once and used in both places? This would keep them in sync to avoid accidental changes to message changing behavior